addons/website_forum/data/forum_demo.xml

```diff
@@ -79,7 +77,6 @@ The CMS editor in Odoo web is nice but I prefer Drupal for customization and the
             <field name="user_id" ref="base.user_demo"/>
             <field name="vote">1</field>
         </record>
-        
```

Comment:
I didn't meant to remove the line but to remove the multiple whitespace in it :)
Keep the blank line, it's how this file is defined: blank line + comment + record(s)
Then should be good :+1:

Reply:
@rdeodoo sorry to understand it wrongly, updated again